packages/webdriver/src/index.js

```diff
@@ -54,7 +54,10 @@ export default class WebDriver {
         params.capabilities = response.value.capabilities || response.value
         params.isW3C = isW3CSession(params.capabilities)
 
-        const prototype = Object.assign(getPrototype(params.isW3C, isChromiumSession(params.capabilities)), proto)
+        const isMobile = Boolean(params.capabilities.deviceName || params.capabilities.platformVersion)
```

Comment:
@christian-bromann It makes sense to check for `deviceName` as this is for mobile only, however `platformVersion` is also returned by browsers on desktop, for example in Firefox.
https://github.com/webdriverio/webdriverio/blob/2b8dacc430ff4fa2d7c401e555846c31cfe979ab/packages/webdriver/tests/__fixtures__/geckodriver.response.json#L18

Couldn't we re-use the same check as for Appium to distinguish between mobile/desktop?
https://github.com/webdriverio/webdriverio/blob/2b8dacc430ff4fa2d7c401e555846c31cfe979ab/packages/webdriver/src/utils.js#L198